src/v/cluster/tx_gateway_frontend.h

```diff
@@ -104,6 +104,10 @@ class tx_gateway_frontend final
           features::feature::transaction_ga);
     }
 
+    bool is_fetch_tx_supported() {
```

Comment:
didn't understand this change.

Reply:
it introduces a feature flag which we use later